crypto/fipsmodule/aes/asm/aesni-x86.pl

```diff
@@ -153,17 +153,19 @@ sub aesni_generate1	# fully unrolled loop
 	&cmp		($rounds,11);
 	&jb		(&label("${p}128"));
 	&lea		($key,&DWP(0x20,$key));
-	&je		(&label("${p}192"));
-	&lea		($key,&DWP(0x20,$key));
```

Comment:
Why remove the lea here? (I remember that I made a similar change to this before where I removed an extra instruction and it actually resulted in a failure; the failure wasn't caught on travis CI because Travis CI's configuration caused the CPUID-based dispatching logic to choose a different code path when running the tests.)

Reply:
Resolved [here](https://github.com/briansmith/ring/pull/707/files#diff-15fb5a9556ee9b43d6a1db688b6ac1b8R810).